tests/entrypoints/llm/test_prompt_logprobs.py

```diff
@@ -0,0 +1,20 @@
+from vllm import LLM, SamplingParams
+
```

Comment:
- Please move this to the logprobs test directory rather than entrypoints
- Do we have a test that this is giving the right answer?